public/static/app/perf-bb/outliers.html

```diff
@@ -0,0 +1,38 @@
+<div class="toolbar">
+    <div class="control-group">
+        <span class="legend">Mode:</span>
+        <label class="radio-inline" ng-repeat="item in outvm.mode.options">
+            <input type="radio" ng-model="outvm.mode.value" value="[[item.id]]" ng-change="outvm.modeChanged()" name="mode" />
+            [[item.name]]
+        </label>
+    </div>
+    <div class="control-group">
+        <span class="legend">Actions:</span>
+        <div class="btn-group" ng-repeat="action in outvm.actions">
```

Comment:
Currently there are no actions - is this intentional?

Reply:
I've added placeholders for them. We will be adding the actions in future tickets.